web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -238,12 +238,14 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
             boolean pdfViewerNeeded = false;
 
             if (!imageViewerNeeded) {
+                String pdfMimeTypeRegx = "application/(x-)?pdf";
```

Comment:
Might be good to have all the mimetype regex patterns as constants, possibly down in AccessCopiesService